### DISCUSSION TITLE
Should Suzie's Comfyroll Studio extension be replaced by a fork?

### DISCUSSION THREAD

**Discussion by andreszs** [Comfy-Org/ComfyUI-Manager#1351] [General]
The original [Comfyroll Studio](https://github.com/Suzie1/ComfyUI_Comfyroll_CustomNodes) developer has vanished almost a year ago, not only from Github but from all networks.

Considering this is an extremely popular extension, should **ComfyUI Manager** step in and replace it by a forked version? I believe this has been done with other extensions, but I'm not sure how long should we wait to take that action. In the context of ComfyUI and SD, a full year is way too much to wait for anyone.

Note that I'm not a fan of that extension, actually I've barely used its nodes but surely many other users rely on that extension for their workflows.

**Comment by ltdrdata** [comment]
This repository is no longer active, so if someone wants to take over maintenance, I will replace it.
